mobile/src/components/course/course.tsx

```diff
@@ -127,19 +150,28 @@ const Course: FC = () => {
         )}
         data={courseModules}
         keyExtractor={({ id }): string => id.toString()}
-        renderItem={({ item: module, index }): ReactElement => (
-          <Pressable
-            onPress={(): void => handleModulePress(module.courseId, module.id)}
-            disabled={!user}
-            style={Boolean(index) && styles.separator}
-          >
-            <Module
-              index={index}
-              title={module.title}
-              description={module.description}
-            />
-          </Pressable>
-        )}
+        renderItem={({ item: module, index }): ReactElement =>
+          tasksLoading ? (
+            <Spinner />
+          ) : (
```

Comment:
Why do we need spinner for each module? If we were loading task only for specific module that yes, but in this case I would render one spinner for all modules